util-core/src/test/scala/com/twitter/concurrent/AsyncStreamTest.scala

```diff
@@ -389,12 +389,14 @@ class AsyncStreamTest extends FunSuite with GeneratorDrivenPropertyChecks {
       }
     }
 
-    test(s"$impl: buffer() works like Seq.splitAt") {
-      forAll { (items: List[Char], bufferSize: Int) =>
-        val (expectedBuffer, expectedRest) = items.splitAt(bufferSize)
-        val (buffer, rest) = await(fromSeq(items).buffer(bufferSize))
-        assert(expectedBuffer == buffer)
-        assert(expectedRest == toSeq(rest()))
+    test(s"$impl: buffer() works like Seq.splitAt for positive values") {
```

Comment:
Not sure what to do for negative values, see also https://github.com/scala/bug/issues/11482